epsagon/events/dbapi.py

```diff
@@ -120,7 +120,12 @@ def __init__(
                 (operation == 'select') or
                 (not trace_factory.metadata_only)
         ):
-            self.resource['metadata']['Query'] = query[:MAX_QUERY_SIZE]
+
+            query_max = query[:MAX_QUERY_SIZE]
```

Comment:
Because we want to conditionally obfus the maxed query, and not the other way around.
I think it should only touch self.resource[meta][query] after the entire query is prepared

Reply:
I meant keeping the same logic but just using the same variable name as query_max is a redundant var (anyway, its not important)

Reply:
I understand now. Definitely agree, will update